docs/core/tutorials/debugging-with-visual-studio.md

```diff
@@ -86,7 +86,7 @@ Your program displays the string that the user enters. What happens if the user
 
 To set a conditional breakpoint and test what happens when the user fails to enter a string, do the following:
 
-# [C#](#tab/csharp)
+#### [C#](#tab/csharp/)
```

Comment:
The tab spec says to use a single `#` symbol. This was likely changed merely to suppress a Markdownlint warning.